### DISCUSSION TITLE
Trame Application as a standalone HTML

### DISCUSSION THREAD

**Discussion by mirabdullahyaser** [Kitware/trame#35] [Q&A]
Hello,

We wanted to use the trame application as a standalone HTML inside our application. I wanted to know what javascript packages or any other front-end libraries are you using to interact with the images like zoom, move, rotate when we are using remote rendering. if you are using your own code of js to view and interact with images then can you provide us with that so we can place that js inside our application and then can run trame server inside our application and then the trame application can be used as an HTML inside our application.

Thank you.

**Comment by jourdain** [comment]
I'm not sure to understand what you are asking, but the client side set of static files are available from [this archive](https://pypi.org/project/pywebvue/#files) under `./src/pywebvue/www/**`. It is also  and published [on npm](https://www.npmjs.com/package/py-web-vue). There will be other modules as well that will need to be exposed depending on which trame element you are using such as plotly, matplotlib, markdown but vuetify and vtk/paraview are definitely built-in...

HTH
